src/GitHub.VisualStudio/Properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,8 @@
 ﻿using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 [assembly: AssemblyTitle("GitHub.VisualStudio")]
 [assembly: AssemblyDescription("GitHub for Visual Studio VSPackage")]
-[assembly: Guid("fad77eaa-3fe1-4c4b-88dc-3753b6263cd7")]
\ No newline at end of file
+[assembly: Guid("fad77eaa-3fe1-4c4b-88dc-3753b6263cd7")]
+[assembly: InternalsVisibleTo("UnitTests")]
```

Comment:
Unfortunately, this causes problems here internally because we sign our official releases of the extension and so `InternalsVisibleToFails`. I think the best thing to do here is just make the class you need access to (``PkgCmdIDList`) public.